src/Containers/MarketPage/MarketPage.scss

```diff
@@ -22,21 +26,40 @@
 .find-vendors-button{
   font-size: 1.5em;
   border-radius: 20px;
+  border: 2px solid khaki;
+  color: cornsilk;
+  letter-spacing: 1px;
   padding: 5px;
   min-width: 100px;
   cursor: pointer;
   text-align: center;
-  margin-top: 30px;
+  // margin-top: 30px;
   outline: 0;
   width: 50%;
-  background-color:cadetblue;
+  background-color:transparent;
   cursor: pointer;
+  margin-top: 50px;
 }
 .find-vendors-button:hover,
 .find-vendors-button:focus {
   transform: scale(1.05);
 }
 
+h3 {
+  letter-spacing: 1px;
+}
+
+.direction-link {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+a:link {
```

Comment:
this link is very difficult to ready on this page - I also think that having the opaque background here might not be the best. I think it would be better darker, but with that background image being so busy it looks like a lot when right up against the google maps.